src/main/typescript/components/form/Validation.ts

```diff
@@ -200,53 +200,25 @@ export function isDaiValid(daiInput: string): boolean {
     return false
 }
 
-export const validateSchemedValue: (schemedValues: SchemedValue[]) => SchemedValue[] = schemedValues => {
-    return schemedValues.map(id => {
-        const nonEmptyScheme = checkNonEmpty(id.scheme)
-        const nonEmptyValue = checkNonEmpty(id.value)
-
-        const idError: SchemedValue = {}
-
-        if (!nonEmptyScheme && nonEmptyValue)
-            idError.scheme = "No scheme given"
-
-        if (nonEmptyScheme && !nonEmptyValue)
-            idError.value = "No identifier given"
+function validateIdentifier(identifierSettings: IdentifiersDropdownListEntry[], schemedValue: SchemedValue): SchemedValue {
+    console.log("validate identifier", schemedValue)
```

Comment:
No validation for `SchemdValue`s like license, types and formats?

Reply:
No, these are not mandatory and they are implemented as single fields in `easy-deposit-ui`. They only get converted to `SchemedValue`s in the background.